Hearthstone Deck Tracker/Windows/MainWindow/MainWindow_Import.cs

```diff
@@ -198,9 +198,10 @@ public partial class MainWindow
 					if(deck != null)
 					{
 						SetNewDeck(deck);
-						if(Config.Instance.AutoSaveOnImport)
-							SaveDeckWithOverwriteCheck();
-					}
+                        if (Config.Instance.AutoSaveOnImport)
+                            SaveDeckWithOverwriteCheck();
+                                                   
+                    }
```

Comment:
Please change indention to tabs, that removes unnecessary changes like this one.